invokeai/app/invocations/image.py

```diff
@@ -375,6 +375,11 @@ class ImageResizeInvocation(BaseInvocation):
     width: int = InputField(default=512, ge=64, multiple_of=8, description="The width to resize to (px)")
     height: int = InputField(default=512, ge=64, multiple_of=8, description="The height to resize to (px)")
     resample_mode: PIL_RESAMPLING_MODES = InputField(default="bicubic", description="The resampling mode")
+    metadata: CoreMetadata = InputField(
+        default=None,
+        description=FieldDescriptions.core_metadata,
+        ui_hidden=True,
+    )
```

Comment:
Wrong merge conflict resolve?

Reply:
No. While scaling down on canvas, the final node is the resize node so thought it was the best place to collect the metadata. Maybe just using the Latents to Image will be the same thing too. I'll take another look when I wake up and decide what to do with this.